konduit-serving-build/src/main/java/ai/konduit/serving/build/cli/BuildCLI.java

```diff
@@ -180,6 +181,11 @@ public void exec(String[] args) throws Exception {
         if(config != null){
             Map<String,String> props = new HashMap<>();
             for(String s : config){
+                if (RemoteUtils.isUrl(s))
+                    continue;
+                if (s.startsWith("http://")) {
+                    s = RemoteUtils.configFromHttp(s);
+                }
```

Comment:
Not sure on the `if(RemoteUtils.isUrl(s))` part. We're skipping URLs but then downloading them on the next section?

Also we should probably use URIResolver.getFile here instead.